repos/fdbt-site/src/pages/globalSettings.tsx

```diff
@@ -0,0 +1,106 @@
+                                        >
+                                            Service day end
+                                        </a>
+                                    </li>
+
+                                    <li className="app-subnav__section-item">
+                                        <a
+                                            className="app-subnav__link govuk-link govuk-link--no-visited-state govuk-link--no-underline"
+                                            href="/components/button/"
+                                        >
+                                            Purchase methods
+                                        </a>
+                                    </li>
+
+                                    <li className="app-subnav__section-item">
+                                        <a
+                                            className="app-subnav__link govuk-link govuk-link--no-visited-state govuk-link--no-underline"
+                                            href="/components/character-count/"
+                                        >
+                                            Time restrictions
+                                        </a>
+                                    </li>
+
+                                    <li className="app-subnav__section-item">
+                                        <a
+                                            className="app-subnav__link govuk-link govuk-link--no-visited-state govuk-link--no-underline"
+                                            href="/components/checkboxes/"
+                                        >
+                                            Multi-operator groups
+                                        </a>
+                                    </li>
+
+                                    <li className="app-subnav__section-item">
+                                        <a
+                                            className="app-subnav__link govuk-link govuk-link--no-visited-state govuk-link--no-underline"
+                                            href="/components/cookie-banner/"
+                                        >
+                                            Travel Zones
+                                        </a>
+                                    </li>
+                                </ul>
+                            </nav>
+                        </div>
+                    </div>
+                    <div className="govuk-grid-column-two-thirds">
+                        <h1 className="govuk-heading-xl">Operator Settings</h1>
+                        <p className="govuk-body">This is a paragraph inside a two-thirds wide column</p>
+                    </div>
+                </div>
+            </main>
+        </div>
+    </BaseLayout>
+);
+
+export const getServerSideProps = (ctx: NextPageContextWithSession): { props: {} } => {
+    if (process.env.STAGE === 'prod' && ctx.res) {
```

Comment:
Why the check for ctx.res here?

Reply:
Because it can be undefined - we do it on other pages where we do the redirect.